core/pkg/server/handler.go

```diff
@@ -167,43 +167,27 @@ func (h *Handler) Do(inChan <-chan *spb.Record) {
 		h.handleRecord(record)
 
 		switch record.RecordType.(type) {
-		case *spb.Record_Alert:
-			h.fwdRecord(record)
-		case *spb.Record_Artifact:
-			h.fwdRecord(record)
-		case *spb.Record_Config:
-			h.fwdRecord(record)
-		case *spb.Record_Files:
-			h.fwdRecord(record)
-		case *spb.Record_History:
-			h.fwdRecord(record)
-		case *spb.Record_Metric:
-			h.fwdRecord(record)
-		case *spb.Record_Output:
-			h.fwdRecord(record)
-		case *spb.Record_OutputRaw:
-			h.fwdRecord(record)
-		case *spb.Record_Preempting:
-			h.fwdRecord(record)
-		case *spb.Record_Stats:
-			h.fwdRecord(record)
-		case *spb.Record_Summary:
-			h.fwdRecord(record)
-		case *spb.Record_Telemetry:
-			h.fwdRecord(record)
-		case *spb.Record_UseArtifact:
+		default:
 			h.fwdRecord(record)
 
+		// Exceptions to the default of forwarding:
 		case *spb.Record_Exit:
+			// The Runtime field is updated on the record before forwarding,
+			// and it is forwarded with AlwaysSend and if syncing Local.
 		case *spb.Record_Final:
+			// Deprecated.
 		case *spb.Record_Footer:
+			// Deprecated.
 		case *spb.Record_Header:
+			// The record's VersionInfo gets modified before forwarding.
 		case *spb.Record_NoopLinkArtifact:
+			// Deprecated.
 		case *spb.Record_Tbrecord:
+			// Never forwarded.
```

Comment:
wait why?

Reply:
i see that in the python code we do forward it

Reply:
tho, it looks like we don't do anything in the sender with the record 🤷

Reply:
yeah, it's just not used. I'll double check that it's OK to not forward it and then update in a different PR (I have more coming for Request records)

Reply:
ooh, glad I checked actually, it's not safe to forward unless we add the corresponding case in the Sender

Reply:
yep, i saw it is just noop in the sender in the python code

Reply:
it's kinda weird in that case, seems that it should be a request and not a record

Reply:
i agree! surfacing these issues is the point of the refactor :) now it's easy to see what the next cleanup steps are